link_folder.go

```diff
@@ -20,16 +22,72 @@ func (c *Client) ListChildren(ctx context.Context, shareID, linkID string) ([]Li
 	return res.Links, nil
 }
 
-func (c *Client) CreateFolder(ctx context.Context, shareID string, req CreateFolderReq) (CreateFolderRes, error) {
+func (c *Client) TrashChildren(ctx context.Context, shareID, linkID string, childIDs ...string) error {
 	var res struct {
-		File CreateFolderRes
+		Responses struct {
+			Responses []struct {
+				LinkResponse struct {
+					LinkID   string
+					Response APIError
+				}
+			}
+		}
 	}
 
-	if err := c.do(ctx, func(r *resty.Request) (*resty.Response, error) {
-		return r.SetResult(&res).SetBody(req).Post("/drive/shares/" + shareID + "/folders")
-	}); err != nil {
-		return CreateFolderRes{}, err
+	for _, childIDs := range xslices.Chunk(childIDs, 150) {
+		req := struct {
+			LinkIDs []string
+		}{
+			LinkIDs: childIDs,
+		}
+
+		if err := c.do(ctx, func(r *resty.Request) (*resty.Response, error) {
+			return r.SetBody(req).SetResult(&res).Get("/drive/shares/" + shareID + "/folders/" + linkID + "/trash_multiple")
+		}); err != nil {
+			return err
+		}
+
+		for _, res := range res.Responses.Responses {
+			if res.LinkResponse.Response.Code != SuccessCode {
+				return fmt.Errorf("failed to import message: %w", res.LinkResponse.Response)
+			}
+		}
+	}
+
+	return nil
+}
+
+func (c *Client) DeleteChildren(ctx context.Context, shareID, linkID string, childIDs ...string) error {
+	var res struct {
+		Responses struct {
+			Responses []struct {
+				LinkResponse struct {
+					LinkID   string
+					Response APIError
+				}
+			}
+		}
+	}
+
+	for _, childIDs := range xslices.Chunk(childIDs, 150) {
+		req := struct {
+			LinkIDs []string
+		}{
+			LinkIDs: childIDs,
+		}
+
+		if err := c.do(ctx, func(r *resty.Request) (*resty.Response, error) {
+			return r.SetBody(req).SetResult(&res).Get("/drive/shares/" + shareID + "/folders/" + linkID + "/delete_multiple")
+		}); err != nil {
+			return err
+		}
+
+		for _, res := range res.Responses.Responses {
+			if res.LinkResponse.Response.Code != SuccessCode {
+				return fmt.Errorf("failed to import message: %w", res.LinkResponse.Response)
```

Comment:
`failed to delete children: %w`?

Reply:
Thanks, copy paste mistake :) fixed in latest PR